apps/api/src/controllers/v1/types.ts

```
@@ -0,0 +1,1018 @@
+  return {
+    scrapeOptions: scrapeOptions.parse({
+      formats: [
+        (pageOptions.includeMarkdown ?? true) ? ("markdown" as const) : null,
+        (pageOptions.includeHtml ?? false) ? ("html" as const) : null,
+        (pageOptions.includeRawHtml ?? false) ? ("rawHtml" as const) : null,
+        (pageOptions.screenshot ?? false) ? ("screenshot" as const) : null,
+        (pageOptions.fullPageScreenshot ?? false)
+          ? ("screenshot@fullPage" as const)
+          : null,
+        extractorOptions !== undefined &&
+        extractorOptions.mode.includes("llm-extraction")
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)